creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -921,7 +933,7 @@ class SnapbackSM {
     this.syncDeDuplicator.removeSync(syncType, userWallet, secondaryEndpoint)
 
     // primaryClockValue is used in additionalSyncIsRequired() call below
-    const primaryClockValue = (await this.getUserPrimaryClockValues([userWallet]))[userWallet]
+    const primaryClockValue = (await this.getUserPrimaryClockValues([userWallet]))[userWallet] || -1
```

Comment:
this probably isn't necessary anymore with the updated fn right? we will default to -1 if a value is not found

Reply:
correct it isn't necessary

Reply:
then we probably don't need this? I think the updated fn should cover this scenario

Reply:
yeah you right, removed